server/src/main/java/org/schoellerfamily/pathfinderhelper/runner/CharacterCommandLineRunner.java

```diff
@@ -0,0 +1,28 @@
+package org.schoellerfamily.pathfinderhelper.runner;
+
+import org.schoellerfamily.pathfinderhelper.datamodel.Character;
+import org.schoellerfamily.pathfinderhelper.repository.CharacterRepository;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Stream;
+
+@Component
+public class CharacterCommandLineRunner implements CommandLineRunner {
+
+    private final CharacterRepository repository;
+
+    public CharacterCommandLineRunner(CharacterRepository repository) {
+        this.repository = repository;
+    }
+
+    @Override
+    public void run(String... strings) throws Exception {
+        // Top beers from https://www.beeradvocate.com/lists/top/
+        Stream.of("Kentucky Brunch Brand Stout", "Good Morning", "Very Hazy", "King Julius",
+                "Budweiser", "Coors Light", "PBR").forEach(name ->
+                repository.save(new Character())
```

Comment:
The current behavior saves a bunch of characters with no attributes set. I don't think that's what you want. ;-) This class and method should be clarified as a demo kind of thing. It is loading a bunch of data on startup.